doc/README.md

```diff
@@ -1,42 +0,0 @@
-# Documentation for scikit-learn
-
-This directory contains the full manual and web site as displayed at
-http://scikit-learn.org.
```

Comment:
I wonder if we should leave these first three lines, plus "see http://...#contribute-documentation"

Reply:
Sounds like a good idea to me